Readme.md

```diff
@@ -96,23 +130,30 @@ compile (for instance with Microsoft MSVC) you need to manually
 set/unset some switches in the compiler command line; see file *types.h*
 for a quick reference.
 
-### Resource For Understanding the Code Base
 
-* [Chess Programming Wiki](https://www.chessprogramming.org/Main_Page) 
-has good overall chess engines explanations 
-(techniques used here are well explained like hash maps etc), it was 
-also recommended by the [support team at stockfish.](http://support.stockfishchess.org/discussions/questions/1132-how-to-understand-stockfish-sources)
+## Resource For Understanding the Code Base and participating in the project
+
+Stockfish's improvement of the last couple of years have been a great community effort. You can help getting stronger in different ways.
+
+### Donating hardware
+
+Improving Stockfish requires a massive amount of testing. You can simply donate your hardware by installing the [Fishtest Worker](https://github.com/glinscott/fishtest/wiki/Running-the-worker) and view the current tests on [Fishtest](http://tests.stockfishchess.org/tests).
+
+### Improving the code
+
+If you want to help to improve the code, there are several valuable ressources to read:
+
+* [In this wiki,](https://www.chessprogramming.org) many techniques used in Stockfish are explained with a lot of background information.
+
+* [The section on Stockfish](https://www.chessprogramming.org/Stockfish) describes many features and techniques used by Stockfish. However, it is generic rather than being focused on Stockfish's precise implementation. Nevertheless, a helpful resource.
 
-* [Here](https://www.chessprogramming.org/Stockfish) you can find a set 
-of features and techniques used by Stockfish and each of them is explained 
-at the wiki, however, it's a generic way rather than focusing on Stockfish's 
-own implementation, but it will still help you.
+* The latest source can always be found on  [GitHub](https://github.com/official-stockfish/Stockfish). Discussions about Stockfish take place in the [FishCooking](https://groups.google.com/forum/#!forum/fishcooking) group and engine testing is done on [Fishtest](http://tests.stockfishchess.org/tests). If you want to help improve Stockfish, please read this [guideline](https://github.com/glinscott/fishtest/wiki/Creating-my-first-test) first, where the fundamentals about Stockfish development are explained.
```

Comment:
```suggestion
* The latest source can always be found on  [GitHub](https://github.com/official-stockfish/Stockfish). Discussions about Stockfish take place in the [FishCooking](https://groups.google.com/forum/#!forum/fishcooking) group and engine testing is done on [Fishtest](http://tests.stockfishchess.org/tests). If you want to help improve Stockfish, please read this [guideline](https://github.com/glinscott/fishtest/wiki/Creating-my-first-test) first, where the basics of Stockfish development are explained.
```

Reply:
thanks for your suggestions. i implemented them